pkg/logging/logging.go

```diff
@@ -17,6 +17,7 @@ const (
 	ConstraintViolations = "constraint_violations"
 	ResourceGroup        = "resource_group"
 	ResourceKind         = "resource_kind"
+	ResourceLabels       = "resource_labels"
```

Comment:
can we also add this to https://github.com/open-policy-agent/gatekeeper/blob/master/website/docs/audit.md#audit-logs so it can be included in the website after the next release.

Reply:
Done